http_check/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import pytest
+from mock import patch
+
+from .common import HERE
+
+
+@pytest.fixture
+def aggregator():
+    from datadog_checks.stubs import aggregator
+    aggregator.reset()
+    return aggregator
+
+
+@pytest.fixture(scope='session')
+def http_check():
+    from datadog_checks.http_check import HTTPCheck
+
+    # Patch the function to return the certs located in the `tests/` folder
+    patcher = patch('datadog_checks.http_check.http_check.get_ca_certs_path', new=mock_get_ca_certs_path)
+    patcher.start()
+
+    http_check = HTTPCheck('http_check', {}, {})
+    yield http_check
```

Comment:
The original tests used a single instance of `HTTPCheck` for every test. I've replicated that by passing the same instance of `HTTPCheck` as a fixture.

Reply:
![image](https://user-images.githubusercontent.com/7241/40646621-01bb704c-632a-11e8-80a9-43e86cc2338e.png)
Let the pytest flow